client/cypress/integration/pages/flight.spec.js

```diff
@@ -19,8 +19,8 @@ describe("check flight page", () => {
     cy.get("#mapContainer");
   });
 
-  it("check presence of barogramm", () => {
-    cy.get("#flight-barogramm");
+  it("check presence of position flight stats", () => {
+    cy.get("#positionStatsTable").find("td");
```

Comment:
wieso das find("td")? Das find dient noch als Selektor die Assertion würde man mit should machen.

cy.get("#positionStatsTable").should("exist");